spec-tests/src/main/java/org/candlepin/spec/bootstrap/data/util/StringUtil.java

```diff
@@ -21,21 +21,114 @@ public final class StringUtil {
 
     private static final Random RANDOM = new Random();
 
+    /** Default length of randomly generated strings */
+    public static final int DEFAULT_RANDOM_STRING_LENGTH = 8;
+
+    /** Character set consisting of upper- and lower-case alphabetical ASCII characters */
+    public static final String CHARSET_ALPHABETICAL = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
+
+    /** Character set consisting of numeric ASCII characters */
+    public static final String CHARSET_NUMERIC = "0123456789";
+
+    /** Character set consisting of characters representing base-16 (hexidecimal) values (0-9, A-F) */
+    public static final String CHARSET_NUMERIC_HEX = "0123456789ABCDEF";
+
+    /** Character set consisting of numeric and upper- and lower-case alphabetical ASCII characters */
```

Comment:
Extra '-' after upper

Reply:
The hyphen connects to "case" following "lower-", and this is shorthand for the following: "upper-case and lower-case" or "upper[-]/lower-case"

Reply:
ooh that's my bad!